client/app/views/steps/password.coffee

```diff
@@ -1,14 +1,24 @@
 StepView = require '../step'
+timezones = require '../../lib/timezones'
+
 
 module.exports = class PasswordView extends StepView
+
     template: require '../templates/view_steps_password'
 
     events:
-        'click button': 'onSubmit'
+        'click button': 'doSubmit'
+
+
+    getDataFromDOM: ->
+        # TODO: get user info
+        # from server
+        return {
+            password: @$('input[name=password]').val()
+            onboardedSteps: ['welcome', 'agreement', 'password']
```

Comment:
No need to change but `onboardedSteps` property is not really retrieved from DOM, so it should not be here or the function may be renamed.